[L'Empire des Cerisiers V1.2-3] Fix minor bug

MonkeyEye opened the pull request:
Fix : incorrect helper stating 1 Gin equals 1000 mon instead of 100
Fix : Init bonus from field not updating on field level changed
Fix : Init mod from armor incorrect calculation
Updated : renamed number of attack / turn to number of action / turn

## Changes / Comments






## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [x] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/Empire_des_cerisiers/roll20-character-sheets) *Roll20 Internal Use only.*

MonkeyEye commented:
The pull request title include the sheet name and version.

Fixed an incorrect helper stating that 1 Gin equals 1000 mon instead of 100.
Fixed initiative bonus optained from a field which was not updating when the field level changed.
Fixed initiative modificator from armor who was incorrectly calculated.
Updated text "Nombre d'attaque par tour" to "Nombre d'action par tour" which translate to "number of attack per turn" to "number of action per turn"

No attributes were changed or removed in the process.
